firefox-ios/Client/Frontend/Settings/Main/Debug/FeatureFlags/FeatureFlagsDebugViewController.swift

```diff
@@ -52,6 +53,21 @@ final class FeatureFlagsDebugViewController: SettingsTableViewController, Featur
         )
     }
 
+    private func getCloseRemoteTabSetting(_ theme: Theme) -> FeatureFlagsBoolSetting {
+        FeatureFlagsBoolSetting(
+            with: .closeRemoteTabs,
+            titleText: NSAttributedString(
+                string: "Enable Close Remote Tabs",
+                attributes: [NSAttributedString.Key.foregroundColor: theme.colors.textPrimary]),
+            statusText: NSAttributedString(
+                string: "Toggle to enable closing tabs remotely feature",
+                attributes: [NSAttributedString.Key.foregroundColor: theme.colors.textPrimary])
+        ) { [weak self] _ in
+            UserDefaults.standard.set(nil, forKey: "\(GleanPlumbMessageStore.rootKey)\("close-tabs-enabled")")
```

Comment:
@skhamis we don't need to set the user defaults here, I had to do it for the microsurvey to reset the expiration that prevented the survey to show more than once as part of the messaging system. 

If there's no additional action needed for the close remote tab setting to trigger it to show / hide, then you don't need to add this.

Reply:
ah gotcha, sounds good -- updated!